test/runtests.jl

```diff
@@ -408,6 +408,10 @@ struct IncompleteInterval <: AbstractInterval{Int} end
             i5 = nextfloat(one(T)) .. 2one(T)
             i_empty = one(T) ..zero(T)
 
+            # - union of multiple intervals
+            # issue #103
+            @test i1 ∪ i2 ∪ i3 ∪ i4 ∪ i5 ∪ i_empty == 0..3
```

Comment:
```suggestion
            @test i1 ∪ i3 ∪ i4 == 0..3  # Equivalent to (i1 ∪ i3) ∪ i4
            @test_throws ArgumentError i1 ∪ i4 ∪ i3  # The order of union matters
            @test ∪(i1, i3, i4) == 0..3
            @test ∪(i1, i4, i3) == 0..3
```
`a + b + c` is parsed as `+(a, b, c)`, but `a ∪ b ∪ c` is not. See https://docs.julialang.org/en/v1.9.3/manual/mathematical-operations/#footnote-2 for the official documentation.

Reply:
The current underlying structure doesn't allow union of disjoint intervals, which leads to `(a ∪ b) ∪ c != a ∪ (b ∪ c)`. I'd suggest the user to use `∪(a, b, c)` in the potential error message.